app/cli.py

```diff
@@ -9,6 +9,8 @@ def cli_runner(config):
     parser.add_argument("--rank-xp-reward", help="Reward per user message")
     parser.add_argument("--announcement-welcome", help="Default welcome message")
     parser.add_argument("--announcement-farewell", help="Default farewell message")
+    parser.add_argument("--badge-veteran-cutoff-date", help="Cutoff date for the \"Operation Elysian Veteran\" badge")
```

Comment:
The cutoff date should not be something that gets modified. It should be a constant somewhere as there's only one date for the cutoff for the "Operation Elysian Veteran" role to be obtained. If another event like r/Place happens in the future, we'd have a different role most likely, so updating then would still not be beneficial.

Reply:
Likewise, others might not have or need a veteran badge, it could just be a Steins;Place thing. If we want to leave functionality for others, then it should be fine here.